libtransmission/torrents.h

```diff
@@ -107,9 +110,22 @@ class tr_torrents
         return std::empty(by_hash_);
     }
 
-    [[nodiscard]] constexpr auto const& sorted_by_id() const noexcept
+    [[nodiscard]] auto get_if(std::function<bool(tr_torrent const*)> pred_in) const
```

Comment:
nit: `get_if` is usually associated with `std::variant`, I dunno, maybe `get_matching` is the name for this.